python/lsst/afw/geom/ellipses/quadrupole.cc

```diff
@@ -70,6 +73,11 @@ PYBIND11_PLUGIN(_quadrupole) {
     clsQuadrupole.def("transformInPlace", [](Quadrupole & self, lsst::afw::geom::LinearTransform const & t) {
        self.transform(t).inPlace();
     });
+    // TODO: pybind11 based on swig wrapper for now. Hopefully can be removed once pybind11 gets smarter
+    // handling of implicit conversions
```

Comment:
Perhaps file a Jira ticket for implementing this upstream? Precisely describing the current and desired behaviours.